packages/client/src/library/Svg/AccordionOpened.js

```diff
@@ -0,0 +1,18 @@
+const AccordionOpened = ({ width = "16", height = "16", className = "" }) => (
```

Comment:
I think we already have CaretDown.js, if it's not the same style let's replace the old one

Reply:
I was thinking about that too, but we are using the CaretDown for dropdown, so I wasn't sure if it's okay to just replace it?

Reply:
Updated!